add_user.py

```diff
@@ -31,9 +15,20 @@ def createUser(username):
 	parser.add_argument('username')
 	args = parser.parse_args()
 
+	name = args.username
+	pw = util.makeCode(12)
+	pw_salt = util.makeCode(8)
```

Comment:
Should we maybe have a constant for this somewhere? So, in the future we won't get passwords with different salt lengths.

Reply:
It's probably actually better for it to be inconsistent. Salts should be considered private data, and having them have varying lengths helps keep them private.